tests/Api/Responses/admin/customer/get_customer_statistics_response.json

```diff
@@ -9,10 +9,11 @@
     "@id": "\/api\/v2\/.well-known\/genid\/@string@",
     "perChannelsStatistics": {
         "@context": "\/api\/v2\/contexts\/Customer",
-        "@id": "\/api\/v2\/admin\/customers",
+        "@id": "\/api\/v2\/admin\/customers\/@string@\/statistics",
         "@type": "hydra:Collection",
         "hydra:member": [
             {
+                "@id": "\/api\/v2\/.well-known\/genid\/@string@",
```

Comment:
Since these statistics aren't resources, I believe, we could remove the ID from serialization, as the channel serves as an identifier here.
```suggestion
```

Reply:
class `CustomerStatistics` doesn't have ID property that i could remove from serialization

Reply:
Have you tried setting up `disable_id_generation_collection`
like its done [here](https://github.com/api-platform/core/blob/main/tests/Fixtures/TestBundle/Entity/DisableIdGeneration.php#L20)?